scale/docs/rest/v6/strike.rst

```diff
@@ -0,0 +1,496 @@
+
+.. _rest_v6_strike:
+
+v6 Strike Services
+==================
+
+These services allow a user to create, view, and manage Strike processes.
+
+.. _rest_v6_strike_list:
+
+**Example GET /v6/strikes/ API call**
+
+Request: GET http://.../v6/strikes/
+
+Response: 200 OK
+
+ .. code-block:: javascript 
+ 
+    { 
+        "count": 3, 
+        "next": null, 
+        "previous": null, 
+        "results": [ 
+            { 
+                "id": 1, 
+                "name": "my-strike-process", 
+                "title": "My Strike Process", 
+                "description": "This is my Strike process for detecting my favorite files!", 
+                "job": { 
+                    "id": 7, 
+                    "job_type": { 
```

Comment:
Let's switch job/job type to the v6 base serializer here. Update every response affected.